src/logic/tokens/utils/formatAmount.ts

```diff
@@ -39,3 +39,11 @@ export const formatAmount = (number) => {
 
   return numberFloat
 }
+
+const options = { style: 'currency', currency: 'USD', minimumFractionDigits: 2, maximumFractionDigits: 8 }
+const numberFormat = new Intl.NumberFormat('en-US', options)
```

Comment:
nitpick: can we name this `usNumberFormatter`? like it's done to the formatters at the top of the file

Reply:
done!